Controller/Advice/Call.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+namespace Riskified\Decider\Controller\Advice;
+
+class Call extends \Riskified\Decider\Controller\AdviceHelper
+{
+
+    /**
+     * Function fetches post data from order checkout payment step.
+     * When 'mode' parameter is present data comes from 3D Secure Payment Authorisation Refuse and refusal details are saved in quotePayment table (additional_data). Order state is set as 'ACTION_CHECKOUT_DENIED'.
+     * In other cases collected payment data are send for validation to Riskified Advise Api and validation status is returned to frontend. Additionally when validation status is not 'captured' order state is set as 'ACTION_CHECKOUT_DENIED'.
+     * As a response validation status and message are returned.
+     *
+     * @return \Magento\Framework\App\ResponseInterface|\Magento\Framework\Controller\Result\Json|\Magento\Framework\Controller\ResultInterface
+     * @throws \Riskified\OrderWebhook\Exception\CurlException
+     * @throws \Riskified\OrderWebhook\Exception\UnsuccessfulActionException
+     */
+    public function execute()
+    {
+        $storeScope = \Magento\Store\Model\ScopeInterface::SCOPE_STORE;
+        $adviseEnabled = $this->scopeConfig->getValue(self::XML_ADVISE_ENABLED, $storeScope);
+        //check whether Riskified Advise is enabled in admin settings
+        if($adviseEnabled == 0){
+            return  $this->resultJsonFactory->create()->setData(['advice_status' => 'disabled']);
+        }
+
+        $params = $this->request->getParams();
+        $quoteId = $this->getQuoteId($params['quote_id']);
+        $quoteFactory = $this->quoteFactory;
+        $quote = $quoteFactory->create()->load($quoteId);
+        //save quote object into registry
+        $this->registry->register($quoteId, $quote);
+
+        $this->api->initSdk();
+        $this->logger->log(sprintf(__('advise_log_json_build'), $quoteId));
```

Comment:
If you're not passing any variables to string, you don't need use sprintf function.